main/src/io/github/iltotore/iron/RefinedTypeOps.scala

```diff
@@ -48,7 +48,10 @@ object RefinedTypeOps:
      */
     type FinalType = T
 
-trait RefinedTypeOpsImpl[A, C, T]:
+
+trait RefinedTypeOpsImpl[A, C, T](using rtcAuto: RuntimeConstraint.AutoDerived[A, A :| C]):
+  given rtc: RuntimeConstraint[A, T] = rtcAuto.inner.asInstanceOf[RuntimeConstraint[A, T]]
```

Comment:
I don't think we need such given instance. If someone want to get the constraint bound to a newtype, it can do it as follow:

```scala
inline def foo[T](using mirror: RefinedTypeOps.Mirror[T], constraint: Constraint[mirror.BaseType, mirror.ConstraintType] /*Or runtime constraint*/) = ???
```

But we can get `rtc` as a public `val` to allow using it in `RefinedTypeOps` extensions like in Cats or ZIO

Reply:
I made it a `val` but also created a `protected given RuntimeConstraint[A, C]`. My reasoning behind having the `protected given` is so that it can be used in the companion object of opaque types when creating codecs etc